javav2/example_code/s3/src/main/java/com/example/s3/S3ObjectOperations.java

```diff
@@ -177,36 +218,50 @@ public static void deleteBucket(S3Client client, String bucket) {
      */
     private static void multipartUpload(String bucketName, String key) throws IOException {
 
-        int MB = 1024 * 1024;
+        int mB = 1024 * 1024;
         // snippet-start:[s3.java2.s3_object_operations.upload_multi_part]
         // First create a multipart upload and get upload id
         CreateMultipartUploadRequest createMultipartUploadRequest = CreateMultipartUploadRequest.builder()
-                .bucket(bucketName).key(key)
+                .bucket(bucketName)
+                .key(key)
                 .build();
+
         CreateMultipartUploadResponse response = s3.createMultipartUpload(createMultipartUploadRequest);
         String uploadId = response.uploadId();
         System.out.println(uploadId);
 
         // Upload all the different parts of the object
-        UploadPartRequest uploadPartRequest1 = UploadPartRequest.builder().bucket(bucketName).key(key)
+        UploadPartRequest uploadPartRequest1 = UploadPartRequest.builder()
+                .bucket(bucketName)
+                 .key(key)
                 .uploadId(uploadId)
                 .partNumber(1).build();
-        String etag1 = s3.uploadPart(uploadPartRequest1, RequestBody.fromByteBuffer(getRandomByteBuffer(5 * MB))).eTag();
+
+        String etag1 = s3.uploadPart(uploadPartRequest1, RequestBody.fromByteBuffer(getRandomByteBuffer(5 * mB))).eTag();
+
         CompletedPart part1 = CompletedPart.builder().partNumber(1).eTag(etag1).build();
 
         UploadPartRequest uploadPartRequest2 = UploadPartRequest.builder().bucket(bucketName).key(key)
                 .uploadId(uploadId)
                 .partNumber(2).build();
-        String etag2 = s3.uploadPart(uploadPartRequest2, RequestBody.fromByteBuffer(getRandomByteBuffer(3 * MB))).eTag();
+        String etag2 = s3.uploadPart(uploadPartRequest2, RequestBody.fromByteBuffer(getRandomByteBuffer(3 * mB))).eTag();
         CompletedPart part2 = CompletedPart.builder().partNumber(2).eTag(etag2).build();
 
 
         // Finally call completeMultipartUpload operation to tell S3 to merge all uploaded

```

Comment:
"S3" -> "Amazon S3" (preferably "Amazon Simple Storage Service (Amazon S3)" on first mention, but probably okay in this particular context.

Reply:
FIxed -